apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/ExplanationOfBenefitE2E.java

```diff
@@ -742,16 +742,19 @@ public void searchEobByPatientIdWithLastUpdatedAndPagination() {
     String firstLink = testUtils.getPaginationLink(response, "first");
     assertTrue(firstLink.contains("_lastUpdated"));
 
+    /* RestAssured will url encode urls by default, and we use pre-encoded urls during pagination, so turn it off
+     * since it will double-encode the lastUpdated field otherwise. */
+    RequestSpecification requestAuthNoEncode = given().spec(requestAuth).urlEncodingEnabled(false);
+
     // Ensure using the next link works appropriately and returns the last 3 results
     given()
-        .spec(requestAuth)
+        .spec(requestAuthNoEncode)
         .expect()
         .body("entry.size()", equalTo(3))
         .body("total", equalTo(expectedTotal))
         .statusCode(200)
         .when()
-        .get(URLDecoder.decode(nextLink));
-    // FUTURE: Decoder is a workaround, fix in https://jira.cms.gov/browse/BFD-2883
+        .get(nextLink);
```

Comment:
Couldn't we simply do this?  Maybe with the comment above `given()`?  Curious what the benefit `requestAuthNoEncode` adds.

```java
 given()
         .spec(requestAuth)
         .urlEncodingEnabled(false)
         .expect()
```